edison/src/test/java/edu/illinois/cs/cogcomp/edison/features/factory/ParsePhraseTypeOnlyTest.java

```diff
@@ -38,23 +42,35 @@ public void setUp() throws Exception {
     public void testParsePhraseCharniak() throws Exception {
         ParsePhraseTypeOnly charniak = ParsePhraseTypeOnly.CHARNIAK;
         assertEquals("[VBD]", charniak.getFeatures(predicate).toString());
-        assertEquals("[NP]", charniak.getFeatures(arg1).toString());
-        assertEquals("[PP]", charniak.getFeatures(arg2).toString());
+        List<String> sortedList = new ArrayList<String>();
+        sortedList.add(charniak.getFeatures(arg1).toString());
+        sortedList.add(charniak.getFeatures(arg2).toString());
+        Collections.sort(sortedList);
```

Comment:
Instead of sorting the features and testing them, it would be better to sort `arg1` and `arg2` establishing an order amongst them. The issue is that the order of items returned by `srlView.getArguments` might not have been the same for Linux vs Mac [Or for different JVM]. It would be better to fix an order for `arg1` and `arg2` in the `setUp()` stage and have the test be consistent.